views/_form.html

```diff
@@ -0,0 +1,18 @@
+<form action="{{action}}" method="post" enctype="multipart/form-data" class="form" novalidate>
```

Comment:
We can start arranging our work a little better. Can this be put in `views/partials` or `views/components` and rename it something like `_basic-form.html`